tools/plop-templates/component-component.hbs

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+export interface {{pascalCase component-name}}Props extends Omit<React.ComponentPropsWithRef<'div'>, 'children'> {
+  children?: React.ReactNode;
+  /**
+   * Documentation the Component API with TsDoc comments in this format.
+   */
+  element?: BoxProps['element'];
+}
```

Comment:
```suggestion
export interface {{pascalCase component-name}}Props extends Omit<React.ComponentPropsWithRef<'div'>, 'children'> {
  children?: React.ReactNode;
  /**
   * Overrides the default element name to apply unique styles with the Customization Provider
   * @default '{constantCase component-name}'
   * @type {BoxProps['element']}
   * @memberof {{pascalCase component-name}}Props
   */
  element?: BoxProps['element'];
}
```

so that we don't have to copy/paste the element description from another package and it stays consistent